src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -2451,7 +2453,7 @@ private void UpdateDatabase(ClusterOperationContext context, string type, Blitta
                         DeleteDatabaseRecord(context, index, items, valueNameLowered, databaseName, serverStore);
                         return;
                     }
-
+                    AssertLicenseLimits(type, serverStore, databaseRecord, items, context);
```

Comment:
You need to add `AssertLicenseLimits` on database creation.
Like we do in 6.0.

Reply:
Done

Reply:
The type here isn't handled.

In 6.0 we go over the relevant types.
https://github.com/ravendb/ravendb/blob/4df04e6692527f7305b656f957cfd07f65bdf944/src/Raven.Server/ServerWide/ClusterStateMachine.cs#L1790

Reply:
Done